blog/content/introtogonum.md

```diff
@@ -0,0 +1,155 @@
+and it is easy to identify the code being executed and examine the implementation.
+Science is built on understanding how an experiment was performed and
+what the results are.
+We believe scientific code should be no different.
+In fact, Gonum is partly distinguished from other libraries in that the entire
+codebase can be run with pure Go -- you only need to know one programming
+language to read Gonum code (that said, it is also easy to call into existing C
+and Fortran libraries).
+This philosophy of simplicity has many follow-on benefits.
+For example, dense one-liners can be hard to understand, creating a barrier for 
+others to confirm correctness, and making future modifications difficult.
+Additionally, one-liners often obscure the necessary operations to execute that
+line of code.
+This creats a tension between code that looks pretty, and a targeted implementation
+that is efficient.
+In Gonum, there is usually only "one way to do things", creating a uniform style
+that in turn makes it easy to read algorithm implementations.
+
+The use of Go as a base language has many benefits beyond simplicity.
+Go is fast, approaching the speed of C.
+Go's language primitives are great for parallel computing, and in fact Gonum
+code can often be made to run in parallel by setting a flag.
+Go is designed for ["programming in the large"](https://talks.golang.org/2012/splash.article),
+a huge benefit for building an ecosystem of libraries that work with and build
+on top of one another.
+Go has extremely legible compile errors, full stack traces, trivial package downloading,
+automatic makefiles and import resolution, code formatting tools, easy code documentation,
+a race detector, and the list goes on.
+
+We believe in a new foundation for scientific computing, and we hope you'll join us.
+Please check back for more discussion on the philosophy of Gonum and the design
+decisions for specific packages.
+
+## Getting started with Gonum
+### Downloading the libraries
+First, you need to [install Go](https://golang.org/doc/install) if it is not
+already installed on your machine.
+It is recommended that you use the most recent version of Go.
+Run 
+```
+go get -u -t gonum.org/v1/gonum
+```
+to download the main set of Gonum packages.
+Then run
+```
+go get -u -t gonum.org/v1/plot
+```
+to download the plotting package.
+
+That's it!
+
+You can test that everything is working properly with 
+```
+cd ~/go/src/gonum.org/v1/gonum  (sub $GOPATH for ~/go if you have $GOPATH set)
+go test ./...
+```
+This will take a few minutes.
+Packages should all have an "ok" or "?" next to them.
+While here, please feel free to browse the source code!
+
+### Learning Go
+The [Go tour](https://tour.golang.org/) is an excellent introduction
+to the Go programming language.
+For quick experiments with the language, the [Go playground](https://play.golang.org/)
+is a very nice sandbox, though unfortunately you cannot import the Gonum librares
+at the present time.
```

Comment:
Removed the caveat. I became less sure it would never change after this thread: https://groups.google.com/forum/#!topic/golang-dev/2x1MbYr8cRs